libkbfs/keybase_daemon_rpc.go

```diff
@@ -402,3 +402,8 @@ func (k *KeybaseDaemonRPC) Shutdown() {
 		k.keepAliveCancel()
 	}
 }
+
+// TeamExit (does not) implement keybase1.NotifyTeamInterface.
+func (k *KeybaseDaemonRPC) TeamExit(context.Context, keybase1.TeamID) error {
+	return nil
```

Comment:
Implementing this as a no-op here since it doesn't seem needed by KBFS for now. Let me know if I'm wrong!

cc @mlsteele

Reply:
Yes good! My bad. It's an unreliable notification anyway, so fine to no-op. If you'd like to bust your cache or something later you could use it for that.

Reply:
Cool; thanks for confirming!

Reply:
Is this right? Jenkins didn't catch this because client doesn't run kbfs. But circle caught it because it revendors keybase1 into kbfs before compiling kbfs?

Also all I ever see when there's a circle failure is this 
![image](https://user-images.githubusercontent.com/705646/31839606-cdc87b7c-b5af-11e7-8c00-0d1b7c32f24c.png)

Reply:
I think circle catches it because on mobile we use gobind, and it freaks out there if interface and implementation don't match.

The "spins forever" thing seems like a CircleCI UI bug; I always just collapsed it and expanded again to see actual error.